google/gce/gce_plugin.rb

```diff
@@ -28,7 +63,7 @@
 
   # This resource was generated using the documentation from https://cloud.google.com/compute/docs/reference/latest/addresses.
   type "address" do
-    href_templates "{{selfLink}}","{{items[*].selfLink}}","{{items.*.addresses[].selfLink}}"
+    href_templates "{{.id}}","{{.items| .[].id}}","{{.items | .[].addresses // [] | .[].id}}"
```

Comment:
@gopinathmahakud1 please revert these to href_templates, that are compatible with all actions. @cdwilhelm  this doesn't need to break existing functionality.